code/game/gamemodes/cult/cult.dm

```diff
@@ -200,6 +201,7 @@
 /datum/game_mode/proc/remove_cultist(datum/mind/cult_mind, show_message = 1)
 	if(cult_mind in cult)
 		cult -= cult_mind
+		cult_mind.current.faction -= "cult"

```

Comment:
"Properly."
This does not actually remove the faction from the list of factions, and from looking around, no faction 'removal' actually seems to remove the faction.